### PR TITLE
Prettier all the things ✨

### PR DESCRIPTION
**Pull request by barrymcgee**

### CODE REVIEW

File: examples/browser/components/statusbar.js [hunk 1]
```diff
@@ -3,20 +3,18 @@
 
 'use strict';
 
-
 const PropTypes = require('prop-types');
 const React = require('react');
 
-const {Col, Row} = require('./widgets');
```
Comment (barrymcgee) [inline examples/browser/components/statusbar.js:10]: These widgets are not being used in this component so I have removed the redundant import.

File: examples/browser/components/statusbar.js [hunk 2]
```diff
@@ -3,20 +3,18 @@
...
-
-
 function StatusBar(props) {
-  const text = props.logs.slice().reverse().join('\n');
-  return <textarea rows="7" readOnly={true} value={text}></textarea>;
+  const text = props.logs
+    .slice()
```
Comment (frankban) [inline examples/browser/components/statusbar.js:11]: This change here is a bit weird, given the original line is not that long. Is there a way for disabling this behavior? If not, nevermind.

File: examples/browser/index.html [hunk 3]
```diff
@@ -13,4 +13,4 @@
   <script src="./index.js"></script>
 </body>
 
-</html>
+</html>
```
Comment (frankban) [inline examples/browser/index.html]: No newline at end of file seems an error.

**Review by frankban — APPROVED** [review]
Looks good with a couple of questions. Also, if we are going to use prettier here, we need to include it in dev dependencies and maybe provide a way for easily running it on the Makefile. Can be a followup branch. Thanks!

**Comment by barrymcgee** [comment]
@frankban 

- Added Prettier as dev dep
- Added pre-commit step to run Prettier on all staged changes
- Fixed newline omission on `index.html` (My editor was responsible)
- No override option available for chained function calls.

**Pull request merged**